lib/renovate.ts

```diff
@@ -8,4 +8,8 @@ proxy.bootstrap();
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 (async (): Promise<void> => {
   process.exitCode = await globalWorker.start();
+  // istanbul ignore if
+  if (process.env.RENOVATE_X_HARD_EXIT) {
```

Comment:
```suggestion
  if (process.env.RENOVATE_EXPERIMENTAL_HARD_EXIT) {
```

Because we're already using `RENOVATE_EXPERIMENTAL_NO_MAVEN_POM_CHECK` ?

Reply:
I think it's pretty long, and I was also basing it off the idea of `X-` fields/headers in other areas of software. I'd prefer to standardize on `RENOVATE_X_` from now on